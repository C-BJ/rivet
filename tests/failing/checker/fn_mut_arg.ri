fn func(mut a: i32) void { }

fn main() void { }
