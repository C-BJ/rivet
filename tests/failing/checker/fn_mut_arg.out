tests/failing/checker/fn_mut_arg.ri:1:12: error: arguments passed by value cannot be mutated
    |
  1 | fn func(mut a: i32) void { }
    |            ^
    = note: only arguments passed by reference can be mutated
rivetc: error: could not compile package `fn_mut_arg`, aborting due to previous error.